variables.virtual.network.tf

```diff
@@ -0,0 +1,137 @@
+variable "create_virtual_network" {
```

Comment:
Consider renaming to virtual_network_creation_enabled - to keep alphabetic order

Reply:
Done